src/owee_debug_line.ml

```diff
@@ -239,3 +239,18 @@ let rec fold_rows header section state f acc =
 
 let fold_rows (header,section) f acc =
   fold_rows header section (initial_state header) f acc
+
+let copy state =
```

Comment:
Abusing functional updates with something like `{ state with address = state.address }` would work as well...
Though being explicit is nice too.